### DISCUSSION TITLE
how to install and use go.data software

### DISCUSSION THREAD

**Discussion by salylu** [WorldHealthOrganization/godata#101] [IT support]
May I ask? I have installed the go.data on my mobile phone to proceed with the toolkit for research.
However, I had some difficulties; as soon I tried to open the mobile software, it required me to sync with HUB.
How may I proceed with HUB?
I need help as this is my first time using this software. 
Thank you so much!
[go.data mobile software.pdf](https://github.com/WorldHealthOrganization/godata/files/10729933/go.data.mobile.software.pdf)

**Comment by sarahollis** [comment]
Hi @salylu thanks for reaching out!

The mobile app does not work in isolation, you must have already installed the webapp and connect the mobile app to that.
You can find some more information on the IT Guide, User Guide, How to Sync Mobile App Guide: https://community-godata.who.int/page/documents

Feel free to ask any further IT questions on our Community of Practice here:  https://community-godata.who.int/
